src/server/getters/get-markets.ts

```diff
@@ -83,19 +85,28 @@ export async function getMarkets(db: Knex, augur: {}, params: t.TypeOf<typeof Ge
 
   let marketsRows = await queryModifier<MarketsContractAddressRow>(db, query, "volume", "desc", params);
 
-  if (params.minInitialRep) {
+  if (params.enableInitialRepFilter) {
     const marketIds = _.map(marketsRows, "marketId");
-    const totalInitialREPStakeRows: Array<TotalInitialREPStakeRow<BigNumber>> = await db.raw(db.raw(`select balance as totalInitialREPStake, markets.marketId from balances join markets on (markets.marketId = balances.owner or markets.initialReporterAddress = balances.owner) join universes on universes.universe = markets.universe where balances.token = universes.reputationToken and markets.marketId in (?)`, [marketIds]).toString());
+    const totalInitialREPStakeRows: Array<TotalInitialREPStakeRow<BigNumber>> = await db.raw(db.raw(`select balance as totalInitialREPStake, markets.marketId, markets.endTime from balances join markets on (markets.marketId = balances.owner or markets.initialReporterAddress = balances.owner) join universes on universes.universe = markets.universe where balances.token = universes.reputationToken and markets.marketId in (?)`, [marketIds]).toString());
     const totalInitialREPStakeByMarket = _.reduce(totalInitialREPStakeRows, (result, row) => {
       if (result[row.marketId]) {
-        result[row.marketId] = result[row.marketId].plus(row.totalInitialREPStake);
+        result[row.marketId].totalInitialREPStake = result[row.marketId].totalInitialREPStake.plus(row.totalInitialREPStake);
       } else {
-        result[row.marketId] = new BigNumber(row.totalInitialREPStake);
+        result[row.marketId] = {
+          totalInitialREPStake:  new BigNumber(row.totalInitialREPStake),
+          endTime: row.endTime,
+        };
       }
       return result;
-    }, {} as {[marketId: string]: BigNumber});
+    }, {} as {[marketId: string]: { totalInitialREPStake: BigNumber, endTime: number }});
     marketsRows = _.filter(marketsRows, (row) => {
-      return totalInitialREPStakeByMarket[row.marketId] && totalInitialREPStakeByMarket[row.marketId].gte(params.minInitialRep || 0);
+      const currentTime = getCurrentTime();
+      const weeksTillV2 = Math.round((V2_CUTOFF_TIMESTAMP - currentTime) / WEEK_IN_SECONDS);
+      let minIntialRep = new BigNumber(0);
+      if (weeksTillV2 <= 12) {
```

Comment:
Could use  STAKE_SCHEDULE.length -1, if i counted correctly there are 13 entries in the collection